github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
Type casting in Go is usually just `int(v)` you don't need to do C style like `(int)(v)`  . It is usually on the other hand better to convert int to int64 and not vice versa. Unless loosing precision is okay.